app/services/participants/base.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "abstract_interface"
+
+module Participants
+  class Base
+    include ProfileAttributes
```

Comment:
I just noticed, without this inclusion, we would not have `ActiveModel::Validations` in this base class. But it refers to errors. Should we include it by default?